### PR TITLE
add CRBN shield

### PR DESCRIPTION
**Pull request by ReFil**
CRBN keymap is identical to planck qmk keymap but for the addition of some Bluetooth stuff
Following advice from innovaker planck keymap will be in a separate pr

### CODE REVIEW

**Comment by innovaker** [comment]
This PR supersedes #421 and #454.

**Comment by innovaker** [comment]
@ReFil, given that this PR supersedes two others and it's been rebased several times, please may I confirm that you've *flashed* and *tested* this PR after all the rebasing?

**Comment by innovaker** [comment]
Oh no, it looks like this shield is still using the deprecated key codes!  Sorry @ReFil, you'll need to upgrade those before we can accept it!  This is the pain of living on the cutting edge.

To make your life easier, you can use this tool which is waiting to be merged:
https://deploy-preview-344--zmk.netlify.app/docs/codes/keymap-upgrader

**Comment by innovaker** [comment]
@ReFil, sorry about this, it looks like another PR conflicted with this one overnight.  Would you mind rebasing it please?  It looks like a simple fix.

**Comment by ReFil** [comment]
Think it's sorted now 😊

File: app/boards/shields/crbn/crbn.keymap [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			>;
+    };
+
+    raise {
+			bindings = <
+				&kp GRAVE &kp N1 &kp N2 &kp N3 &kp N4 &kp N5 &kp N6 &kp N7 &kp N8 &kp N9 &kp N0 &kp BSPC
+				&kp DEL &kp F1 &kp F2 &kp F3 &kp F4 &kp F5 &kp F6 &kp MINUS &kp EQUAL &kp LBKT &kp RBKT &kp BSLH
+				&trans &kp F7 &kp F8 &kp F9 &kp F10 &kp F11 &kp F12 &kp HASH &kp BSLH &kp PG_UP &kp PG_DN &trans
+				&trans &trans &trans &trans &mo 3 &trans &trans &trans &kp C_NEXT &kp C_VOL_DN &kp C_VOL_UP &kp C_PP
+    	>;
```
Comment (innovaker) [inline app/boards/shields/crbn/crbn.keymap:44]:
```suggestion
			>;
```

File: app/boards/shields/crbn/crbn.keymap [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    };
+		control {
+			bindings = <
+				&reset &bootloader &bt BT_CLR &bt BT_PRV &bt BT_NXT &trans &trans &trans &trans &trans &trans &trans
+				&trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+				&trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+				&trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+    	>;
```
Comment (innovaker) [inline app/boards/shields/crbn/crbn.keymap:52]:
```suggestion
			>;
```

File: app/boards/shields/crbn/crbn.overlay [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix-transform.h>
+
+/ {
+	chosen {
+		zmk,kscan = &kscan0;
+
```
Comment (innovaker) [inline app/boards/shields/crbn/crbn.overlay:12]:
```suggestion
```

File: app/boards/shields/crbn/readme.md [hunk 4]
```diff
@@ -0,0 +1 @@
+A 40% Ortho keyboard made by Polarity Works
```
Comment (innovaker) [inline app/boards/shields/crbn/readme.md:1]: It's not essential, but you might want to include a link to your website or repository if you'd like the publicity.

Comment (ReFil) [reply app/boards/shields/crbn/readme.md:1]: We dont have a website yet lol :D

Comment (innovaker) [reply app/boards/shields/crbn/readme.md:1]: I guess https://keygem.store/collections/group-buys/products/group-buy-featherlight-40-kit isn't an option then?

**Pull request merged**